server/core/config/base.py

```
@@ -14,6 +14,11 @@
 from pathlib import Path
 from dotenv import load_dotenv
 
+# cloudinary
+import cloudinary
+import cloudinary.uploader
+import cloudinary.api
```

Ruff (0.8.2)

`cloudinary.api` imported but unused

Remove unused import: `cloudinary.api`

(F401)